### ISSUE TITLE
Add trader data file

### ISSUE THREAD

**Issue by thaddeus** [TarkovTracker/tarkovdata#25]
Add a trader data file for common trader data & localization information

**Comment by thaddeus** [comment]
Essentially looking for this:
```
[
	{
		"id": 0,
		"locale": {
			"en": "Prapor"
		},
		"wiki": "https://escapefromtarkov.gamepedia.com/Prapor",
		"loyalty": [
			{
				"loyaltyLevel": 1,
				"level": 1,
				"rep": 0,
				"sales": 0,
				"salesCur": "rubles"
			},
			{
				"loyaltyLevel": 2,
				"level": 15,
				"rep": 0.2,
				"sales": 75000,
				"slesCur": "rubles"
			}
		]
	},
	{
		"id": 1,
		"locale": {
			"en": "Therapist"
		},
		"wiki": "https://escapefromtarkov.gamepedia.com/Therapist",
		...
	}
]
```

**Comment by thaddeus** [comment]
Issue wasn't closed automatically with the merge of #41 - Thanks for the help @Filodoxia!
